skimage/transform/pyramids.py

```diff
@@ -1,7 +1,7 @@
 import math
 import numpy as np
 from scipy import ndimage as ndi
-from ..transform import resize
+from ._warps import resize
```

Comment:
This appears to be breaking some CI builds... https://dev.azure.com/scikit-image/scikit-image/_build/results?buildId=2282&view=logs&j=bc76316b-1dff-5b05-0ff1-0b85ce9c0f8b&t=7992ab4a-b77a-5ac4-43d7-f576e552c3e1